extension/chrome/elements/compose-modules/compose-input-module.ts

```diff
@@ -78,13 +78,24 @@ export class ComposeInputModule extends ViewModule<ComposeView> {
     return this.view.sendBtnModule.popover.choices.richtext;
   };
 
+  public willInputLimitBeExceeded = (textToPaste: string, targetInputField: HTMLElement) => {
+    const limit = 50000;
+    const currentLength = targetInputField.innerText.length;
+    const isInputLimitExceeded = textToPaste.length + currentLength > limit;
+    return isInputLimitExceeded;
+  };
+
   private handlePaste = () => {
     this.squire.addEventListener('willPaste', (e: WillPasteEvent) => {
       const div = document.createElement('div');
       div.appendChild(e.fragment);
       const html = div.innerHTML;
       const sanitized = this.isRichText() ? Xss.htmlSanitizeKeepBasicTags(html, 'IMG-KEEP') : Xss.htmlSanitizeAndStripAllTags(html, '<br>', false);
       Xss.setElementContentDANGEROUSLY(div, sanitized); // xss-sanitized
+      if (this.willInputLimitBeExceeded(sanitized.trim(), this.squire.getRoot())) {
```

Comment:
Do we need `trim()`? Why? `sanitized` is HTML. Did you notice any extra space appear during testing?

Reply:
Yes, the secure compose has an extra space.

(before)
<img width="851" alt="Screenshot 2023-02-16 at 5 43 23 PM" src="https://user-images.githubusercontent.com/46025304/219328696-1bd676d4-9e4e-4926-8f67-c785b775c5e2.png">

(after)
<img width="827" alt="image" src="https://user-images.githubusercontent.com/46025304/219329924-3d1cb845-e4da-428b-a2c2-49632e2908d1.png">